static/app/locale.spec.tsx

```diff
@@ -49,4 +49,17 @@ describe('locale.gettextComponentTemplate', () => {
       screen.getByText(textWithMarkupMatcher('1st: one2nd: two'))
     ).toBeInTheDocument();
   });
+
+  it('should render multiple groups with the same name', () => {
+    const RenderChildren = ({children}: {children?: React.ReactNode}) => children;
+    render(
+      <div>
+        {tct('[render:one] [render:two] [render:three]', {
+          render: <RenderChildren />,
+        })}
+      </div>
+    );
+
+    expect(screen.getByText(textWithMarkupMatcher('one two three'))).toBeInTheDocument();
```

Comment:
Can you write a test where renderChuldren is a hyml node or react node too, so we can assert something like `<b>one<\b> <b>two<\b>` as well.

Reply:
Sure can do 👍  Though the current one is already a react node. So it should not behave differently.

Reply:
it shouldn't i know. but i'm worried about `<Fragment>` getting rendering instead of the thing that's been passed in. Something that could break one day